make/modules/jdk.jdwp.agent/Lib.gmk

```diff
@@ -70,7 +70,9 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBJDWP, \
 
 $(BUILD_LIBJDWP): $(call FindLib, java.base, java)
 
-TARGETS += $(BUILD_LIBJDWP)
```

Comment:
While this work, this is not normally how we do things in the build.

If you ever want to have this upstreamed, you need to put the entire SetupJdkLibrary inside the ifeq block.

Reply:
Only the `$(eval $(call SetupJdkLibrary, BUILD_LIBJDWP` block or also the
```
$(BUILD_LIBJDWP): $(call FindLib, java.base, java)
TARGETS += $(BUILD_LIBJDWP)
```

?